clouda-inc/bisco-components
# Feature/product detail update

What changes did you make?
Moved inventory available to price break table for product detail page

Why did you make a change?

This improved user experience by positioning inventory information with other buying criteria. 

How do we test?
View Product detail page for part number 1412300. The pricebreak should be visible inside the component. 
We did not test page load impact, please advise. 

Is this a breaking change? 
No, Modifies existing component.
